release_bot/releasebot.py

```diff
@@ -124,12 +139,28 @@ def find_newest_release_pull_request(self):
 
             for edge in reversed(edges):
                 cursor = edge['cursor']
-                match = re.match(r'(.+) release', edge['node']['title'].lower())
-                if match and validate(match[1]):
+                last_version = Version(self.github.latest_release())
+                version = ''
+                match = False
+                re_match = re.match(r'(.+) release', edge['node']['title'].lower())
```

Comment:
I don't we actually need this code here since the release PRs will not have these titles

Reply:
how come?

Reply:
The release pull requests won't be titled with "new major release", they will be titled with "0.4.2 release", still.

Reply:
Ok, I thought you're talking just about  the `re.match` line.